examples/src/QueryExample/QueryExample.cs

```diff
@@ -210,13 +215,13 @@ private async Task InteractiveTx()
                     { "$season_id", YdbValue.MakeUint64(3) }
                 };
 
-                var response = await tx.ReadFirstRow(
+                var response = await tx.ReadScalar(
```

Comment:
need LIMIT 1 in query and check single line result in helper

Reply:
tx.ReadScalar uses ReadScalarHelper, which in turn uses ReadSingleRowHelper, which checks for a single line